app/views/account_invitations/_account_invitation.html.slim

```diff
@@ -1,7 +1,6 @@
 tr
   td= account_invitation.name
   td= account_invitation.email
-  td= account_invitation.account.name
```

Comment:
Why deleted? I think it was ok.

Reply:
No, it wasn't. All invitations belong to the same account which name on top of the table